### PR TITLE
Shortcuts for root commands

### PR DESCRIPTION
**Pull request by elglogins**
Current command names are very explicit and nice to get started with, especially when using for single tenant as part of the CI/CD pipeline.

But, when working with CLI as an alternative to app.enterspeed.com it quickly becomes a lot of typing. My proposal is to introduce shortcuts for root commands.

Such as:
```
tenant -> t
environment -> e
environment-client -> ec
domain -> d
source-group -> sg
source -> s
views -> v
source-entity -> se
schema -> sc
deployment -> dp
```

### CODE REVIEW

**Comment by elglogins** [comment]
I'm not too happy with output of the cli on help, but, I don't see a better way to do it with current implementation?

```
Commands:
  login               Login using OAuth
  tenant              Tenant
  t                   Tenant
  environment         Environment
  e                   Environment
  environment-client  Environment client
  ec                  Environment client
  domain              Domain
  d                   Domain
  source-group        Source group
  sg                  Source group
  source              Source
  s                   Source
  views               Generated views
  v                   Generated views
  source-entity       Source entities
  se                  Source entities
  schema              Schemas
  sc                  Schemas
  deployment
  dp
```

File: src/Enterspeed.Cli/Commands/Deploy/DeployCommands.cs [hunk 1]
```diff
@@ -4,15 +4,13 @@ namespace Enterspeed.Cli.Commands.Deploy
 {
     public class DeployCommands
     {
-        public static Command BuildCommands()
+        public static Command[] BuildCommands()
         {
-            var command = new Command("deployment")
+            return new[] { "deployment", "dp" }.Select(commandName => new Command(commandName)
```
Comment (jthind) [inline src/Enterspeed.Cli/Commands/Deploy/DeployCommands.cs:9]: I don't think this is the right way to add an alias 😄 
Just do `command.AddAlias("dp");`

**Review by jthind — CHANGES_REQUESTED** [review]
Command has `.AddAlias()`
I really like the idea of adding alias

File: src/Enterspeed.Cli/Commands/Deploy/DeployCommands.cs (hunk 1 above)
Comment (elglogins) [reply src/Enterspeed.Cli/Commands/Deploy/DeployCommands.cs:9]: :-o I totally missed it, I looked for it though. Will update

**Comment by elglogins** [comment]
Updated PR according to your comments.

```
Commands:
  login                   Login using OAuth
  t, tenant               Tenant
  e, environment          Environment
  ec, environment-client  Environment client
  d, domain               Domain
  sg, source-group        Source group
  s, source               Source
  v, views                Generated views
  se, source-entity       Source entities
  sc, schema              Schemas
  deployment, dp

```

**Review by jthind — APPROVED** [review]
Nice 👍

**Pull request merged**